lib/views/dialog.ts

```diff
@@ -0,0 +1,37 @@
+import { TextEditor } from "atom"
+
+export default class Dialog {  
```

Comment:
I actually created a dialog view for some other atom packages https://github.com/UziTech/atom-modal-views. Do we want to use that instead of creating our own?

/cc @aminya

Reply:
> I actually created a dialog view for some other atom packages UziTech/atom-modal-views. Do we want to use that instead of creating our own?

That sounds like a good plan! I love reusing code.